tests/security/ima/evm_setup.pm

```diff
@@ -38,7 +38,10 @@ sub run {
 
     add_grub_cmdline_settings("evm=fix ima_appraise=fix ima_appraise_tcb", update_grub => 1);
 
-    power_action('reboot', textmode => 1);
+    record_info("bsc#1189988: ", "We need disable secureboot with ima fix mode");
+    power_action("reboot", textmode => 1);
```

Comment:
Coding style:
`'reboot'`
in order to align with others in the same pm file.

Reply:
record_info("bsc#1189988: ", "We need disable secureboot with ima fix mode");
    power_action("reboot", textmode => 1);

I will keep the current style, since many lines have "" used